pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -560,32 +560,31 @@ def avg_time(cimtime, num_ops):
             number_of_operations = inst['NumberOfOperations']
 
             cimom_avg_elapsesd_time = avg_time(inst['CimomElapsedTime'],
-                                               number_of_operations)
+                                               number_of_operations) / 1000
 
             provider_avg_elapsesd_time = avg_time(inst['ProviderElapsedTime'],
-                                                  number_of_operations)
+                                                  number_of_operations) / 1000
 
             avg_request_size = avg_value(inst['RequestSize'],
                                          number_of_operations)
             avg_response_size = avg_value(inst['ResponseSize'],
                                           number_of_operations)
 
-            rows.append([number_of_operations,
-                         "{:.1f}".format(cimom_avg_elapsesd_time),
-                         "{:.1f}".format(provider_avg_elapsesd_time),
-                         "{:.1f}".format(avg_request_size),
-                         "{:.1f}".format(avg_response_size),
-                         op_type_name])
+            rows.append([op_type_name,
+                         number_of_operations,
+                         "{:.3f}".format(cimom_avg_elapsesd_time),
+                         "{:.3f}".format(provider_avg_elapsesd_time),
+                         "{:.0f}".format(avg_request_size),
+                         "{:.0f}".format(avg_response_size)])
 
-        headers = ['Operation\nCount', 'Server\ntime(usec)',
-                   'Provider\nTime(usec)', 'Request\nsize(bytes)',
-                   'Response\nsize(bytes)',
-                   'Operation Name']
+        headers = ['Operation', 'Count', 'Server Time\n[ms]',
+                   'Provider Time\n[ms]', 'Request Size\n[B]',
+                   'Response Size\n[B]']
 
         context.spinner_stop()
         click.echo(format_table(
             rows, headers,
-            title='WBEM server CIM Operation Statistical Data',
+            title='Server statistics',
```

Comment:
Should we expand this more to be sure the reader understands the table from the results (i.e. server maintained statistics) or something similar?? In this case it is not an issue of realestate on the line since the table itself is much longer than the title.

Reply:
We should use one consistent term and then use it anywhere in the docs and also as the display title. So far, it seems we have not agreed to one single term, I see all kinds of related but different terms in the docs.

**DISCUSSION**: What should our single term be, for client statistics and server statistics?

Reply:
Since both of these entities have server statistics, that is not complete enough.  That it why I have tried to use server maintained/managed statistics vs pywbemcli maintained/managed statistics.
We can standardize to one of those temrs (ex. server maintained statistics vs pywbemcli maintained statistics)

If you have a better term I have no objection.

Reply:
Ok, I understand your argument now. However, I'm not sure I like it.

The client statistics has a server time value (not server statistics). Does that warrant the use of a more complex term for it?

Reply:
I have created PR #914 for the one remaining discussion point about the terms we use. RESOLVED.